besu/src/main/java/org/hyperledger/besu/cli/config/NetworkName.java

```diff
@@ -31,6 +31,9 @@ public enum NetworkName {
   /** LUKSO mainnet network name. */
   LUKSO("/lukso.json", BigInteger.valueOf(42)),
 
+  /** EPHEMERY network name. */
+  EPHEMERY("/ephemery.json", BigInteger.valueOf(39438135)),
```

Comment:
will the network id for ephemery network ALWAYS come from the genesis file? so it might be misleading to have this initial value set here? maybe set it to -1 or 0 so it is obvious this value won't actually be used. if that isn't possible, at least a comment to indicate that.

Reply:
The initial network ID will come from the genesis file and an update made from there. When I ran the node I noticed the network Id that is outputted on the log is the hardcorded one here. That's why I am updating this value alongside the chainId on the Genesis file.

In otherwords the value here will be used to update the chainId. So that the value that gets displayed on the log will match with what is on the genesis file.